lambda-utils/package.json

```diff
@@ -24,37 +24,35 @@
   "author": "Onica Group",
   "homepage": "https://docs.onica.com/projects/sailplane",
   "contributors": [
-    "Adam Fanello <afanello@onica.com>"
+    "Adam Fanello <adam.fanellorackspace.com>"
   ],
   "devDependencies": {
-    "@middy/core": "^1.0.0",
-    "@middy/http-cors": "^1.0.0",
-    "@middy/http-event-normalizer": "^1.0.0",
-    "@middy/http-header-normalizer": "^1.0.0",
-    "@middy/http-json-body-parser": "^1.0.0",
+    "@middy/core": "^2.5.1",
+    "@middy/http-cors": "^2.5.1",
+    "@middy/http-event-normalizer": "^2.5.1",
+    "@middy/http-header-normalizer": "^2.5.1",
+    "@middy/http-json-body-parser": "^2.5.1",
     "@sailplane/logger": "file:../logger",
-    "@types/aws-lambda": "^8.10.51",
-    "@types/jest": "23.3.13",
-    "@types/node": "^10.17.21",
-    "aws-sdk": "^2.675.0",
-    "http-errors": "^1.7.3",
-    "jest": "^23.6.0",
-    "ts-jest": "^23.10.5",
-    "ts-loader": "^2.3.7",
-    "ts-node": "^8.8.1",
-    "typescript": "^2.9.2"
+    "@types/aws-lambda": "^8.10.83",
+    "@types/jest": "26.0.24",
+    "@types/node": "^12.20.21",
+    "http-errors": "^1.8.0",
+    "jest": "^26.6.3",
+    "ts-jest": "^26.5.6",
+    "ts-node": "^10.2.1",
+    "typescript": "3.8.3"
```

Comment:
Was expecting `^3.8.3`.

Reply:
That would result in v3.9.4 being installed and used to build the `.d.ts` files. Hopefully they are backward compatible with all of v3, but forcing v3.8.3 to be used to compile guarantees that the resulting output will be compatible with v3.8.3 and everything forward.

Why v3.8.3? Well there you have me. It's the version used by Angular 9, which I'm using in a project, and of course use the same version for the Lambdas for the sake of code sharing. So it began as totally self serving... some older v3 that was still stable would have been better but require determining just what a good choice is. At this point, a few other sailplane libraries are also locked at v3.8.3 so just sticking with it as the minimum. (Again, hopefully the result is backward compatible to everything v3.x though.)